### DISCUSSION TITLE
Support the "Logitech Speed Force Wheel for Wii" all games like other Wheels?

### DISCUSSION THREAD

**Discussion by karzinogen** [berarma/new-lg4ff#83] [Q&A]
The original Logitech Speed Force Wireless Wheel for Wii supports - as far as I know - only 2 games. Can I use this steering wheel within Linux for all other games?

**Comment by berarma** [comment]
It should work as a generic wheel.

**Comment by karzinogen** [comment]
Thank you (also for your support) - great work.
